tests_tf/test_utils_tf.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+import unittest
+import tensorflow as tf
+
+from cleverhans import utils_tf
+
+
+def numpy_kl_with_logits(q_logits, p_logits):
```

Comment:
It's probably a bit confusing to make q be the left argument, because definitions of the KL divergence almost always use p as the left argument. https://en.wikipedia.org/wiki/Kullback%E2%80%93Leibler_divergence